test_runner/fixtures/pg_version.py

```diff
@@ -25,7 +25,10 @@ class PgVersion(str, enum.Enum):
 
     # Make it less confusing in logs
     def __repr__(self) -> str:
-        return f"'{self.value}'"
+        return f"{self.value}"
+
+    def __str__(self) -> str:
+        return repr(self)
```

Comment:
I think this is the default behaviour

Reply:
That's interesting!
Could you share an example of failed tests, we don't have such failures in `main`, and I don't see any related change in this PR.

Btw, you're right, `str` is not defaulted to `repr` for `PgVersion`.

Reply:
I had the following error:

```FAILED test_runner/regress/test_pg_regress.py::test_pg_regress - FileNotFoundError: [Errno 2] No such file or directory: '/Users/sasha/neon/pg_install/build/vPgVersion.V14/src/test/regress/pg_regress'```

Reply:
I've reverted the changes to `__repr__` and made `__str__` print the expected thing, and added a comment about how this is related to the enum changes in Python 3.11